package/package.json

```diff
@@ -29,13 +29,46 @@
 	},
 	"type": "module",
 	"peerDependencies": {
-		"astro": "^4.0.0"
+		"@vitejs/plugin-react": "^4.2.1",
```

Comment:
why is this one needed and not the others? only for the preamble thing?

Reply:
Yeah that's literally it. But if they aren't using React, then they won't need the import anyways tbh and we can just catch and swallow the error